tests/test_freetextsearch.py

```diff
@@ -51,7 +1364,6 @@ def testSearchFields(self):
         self.assertTrue(response.status_code == 200)
         json_response = json.loads(response.data.decode('utf-8'))
         self.assertEqual(len(json_response['data']), 1)
```

Comment:
Yes when we spoke about it a while ago - it was your suggestion to put it in the fields object and that approached was reviewed, etc... May be i did not understand it right. No i do not think it is more intuitive, but it is easy ;-).  I do not have a strong opinion on this for regular search, but for the bestHitSearch i think there should be a way to disable highlights. At least the way it is being used in the filter by target - it is redundant.